### ISSUE TITLE
 Integrate storage measurement tools into the test framework

### ISSUE THREAD

**Issue by piotr-dziubecki** [casper-network/casper-node#2423]
Follow up after #2346. We need to start taking the on-disk space consumption more seriously. This means we will need to measure not only the performance impact, when making changes to the EE, but also the size:

1. ToBytes size
2. Trie size

You’ve been working on this recently @Michał. Can you build tools/integrate them into the WasmTestBuilder ? We could use that in Dan’s recent PR to measure how much the auction size changes. We did measure the MainNet auction size some time ago but that was only the ToBytes representation and did not take into account the Trie/lmdb overheads.

https://casperlabs-team.slack.com/archives/C02BSLUGUGM/p1638179552157300?thread_ts=1638178274.156400&cid=C02BSLUGUGM

**Comment by dwerner** [comment]
We need to watch data.lmdb and it's growth, so we should add the capacity to check the on-disk size of lmdb during a test. This would be useful to know, for example, the size growth after N transfers, purses, auction steps with N delegators, etc.

I'm requesting to break off the on-disk size into another task. (TODO: add link to task when it's created)

https://casperlabs-team.slack.com/archives/C02FE1UTRS6/p1639067875391300
